workspace/apps/pidp/src/app/app.component.spec.ts

```diff
@@ -32,7 +32,15 @@ describe('AppComponent', () => {
     component = TestBed.inject(AppComponent);
   });
 
-  it('should create the app', () => {
-    expect(component).toBeTruthy();
+  describe('INIT', () => {
+    given('component initializes', () => {
+      when('navigation ends', () => {
+        component.ngOnInit();
+
+        then('should set the title', () => {
+          expect(component.title).toBe('Provider Identity Portal');
```

Comment:
You're testing that a member variable is set, which is hardcoded in the constructor, but you want to check that the TitleService has the correct title based on a route end event triggering setPageTitle().